views/js/controller/PrintTest/render.js

```diff
@@ -102,6 +103,31 @@ define([
         });
     }
 
+    /**
+     * Extract shared stimulus from a item, put it in section before a item
+     * Remove repeated shared stimulus
+     * @param {JQuery} $container
+     */
+    function extractSharedStimulus($container) {
```

Comment:
What would be the behavior in case sharedStimulus includes audio/video?

Reply:
In pdf will be only text from such shared stimulus, instead of the video - an empty place